packages/graphql/src/translate/create-interface-projection-and-params.ts

```diff
@@ -121,7 +121,7 @@ function createInterfaceSubquery({
 
     const param = `${nodeVariable}_${refNode.name}`;
     const relatedNode = new Cypher.NamedNode(param, {
-        labels: [refNode.name], // NOTE: should this be labels?
+        labels: [refNode.getMainLabel()],
```

Comment:
Should not be 
```suggestion
        labels: [refNode.getLabels(context)],
```
instead?

Reply:
Yes that's correct. Glad you point this out!